pages/reference-smart-account/guards/setModuleGuard.mdx

```diff
@@ -0,0 +1,63 @@
+import { Tabs, Callout } from 'nextra/components'
+
+# `setModuleGuard`
+
+Set Module Guard `moduleGuard` for the Safe. Make sure you trust the module guard.
+
+<Callout type='warning'>
+  Set a module guard that checks transactions initiated by the module before
+  execution This action can only be done via a Safe transaction.
+</Callout>
+
+<Callout type='error' emoji='🚨'>
+  Since a module guard has full power to block Safe transaction execution
```

Comment:
I guess "Module Guard" should be written with the first letter uppercase:
"module guard" => "Module Guard".
It's written like this in the definition, but not followed in other places in this page.